packages/web-components/src/components/tabs-extended/tab.ts

```diff
@@ -52,6 +52,23 @@ class DDSTab extends StableSelectorMixin(LitElement) {
     this._index = index;
   }
 
+  protected updated(_changedProperties: Map<string | number | symbol, unknown>): void {
+    if (_changedProperties.has('selected')) {
+      const { eventTabSelected } = this.constructor as typeof DDSTab;
+      this.dispatchEvent(
+        new CustomEvent(eventTabSelected, {
+          bubbles: true,
+          composed: true,
+          detail: {
+            selected: this.selected,
+          },
+        })
+      );
+    }
```

Comment:
Great use of the `dispatchEvent()` method to listen and retrigger the `sameHeight` utility that was giving each of the observed elements `0px` height.